packages/types/migration.d.ts

```diff
@@ -44,7 +44,8 @@ export interface MigrationLoggingParams {
 
 export type ParallelScanMigrationParams = MigrationLoggingParams & DynamoDbParallelScanParams;
 
-export type GranuleMigrationParams = ParallelScanMigrationParams & GranuleDynamoDbSearchParams;
+export type GranuleMigrationParams = ParallelScanMigrationParams &
```

Comment:
The last stanza here could probably be made to be a formal type.  🤔

Reply:
Yeah, it might be more readable as-is though? I think we'd only use a separate type for the migration params in this one spot?

I'm not a TypeScript expert so you may be right and it would be better to have discrete types for everything.

Reply:
I think I'm ok with this one for now.   Particularly as we're removing it in P3.